### DISCUSSION TITLE
Resize Team Logo

### DISCUSSION THREAD

**Discussion by hamboneaf** [vasqued2/ha-teamtracker#131] [Q&A]
I have used the following to adjust the font size and thus the background image adjust automatically. However, I would like to adjust the team logo because it is still too small.  Any help is appreciated.

https://github.com/vasqued2/ha-teamtracker/wiki/Using-CARD-MOD-card-to-format-the-TEAMTRACKER-CARD
 
type: custom:teamtracker-card
entity: sensor.tt_sensor
card_mod:
  style: |
    ha-card {
        font-size: 2.7em; 
        line-height: 2.7em;
    }

**Comment by vasqued2** [comment]
Try something like the following.  You may have to play with the width too.  I don't know HTML to well so there is probably other ways to tweak the logo's size too that I'm just not familiar with.
```
card_mod:
  style: |
    ha-card {
      font-size: 2.7em;
      line-height: 2.7em;
    }
    .logo {
        height: 2.7em;
    }
```
